debezium-bom/pom.xml

```diff
@@ -56,6 +54,7 @@
         <version.jsonassert>1.5.0</version.jsonassert>
         <version.org.reflections>0.9.12</version.org.reflections>
         <version.cassandra.unit>3.5.0.1</version.cassandra.unit>
+        <rest-assured.version>4.3.3</rest-assured.version>
```

Comment:
Gnarf, can we put "version" first in the name, as for other properties? This sort of inconsistency is giving me stress disorders ;)